### PR TITLE
Add more plugins to OPTIONAL_GEMS in haml_lint.rb

### PR DESCRIPTION
**Pull request by yykamei**
Similar to #683

Also, I added [rubocop-sequel](https://github.com/rubocop-hq/rubocop-sequel)
to haml_lint.rb as well as rubocop.rb.

### CODE REVIEW

File: lib/runners/processor/haml_lint.rb [hunk 1]
```diff
@@ -53,6 +53,15 @@ class Processor::HamlLint < Processor
...
+      # The followings are maintained by rubocop-hq. See https://github.com/rubocop-hq
+      GemInstaller::Spec.new(name: "rubocop-rails", version: []),
+      GemInstaller::Spec.new(name: "rubocop-rspec", version: []),
+      GemInstaller::Spec.new(name: "rubocop-minitest", version: []),
+      GemInstaller::Spec.new(name: "rubocop-performance", version: []),
+      GemInstaller::Spec.new(name: "rubocop-rubycw", version: []),
+      GemInstaller::Spec.new(name: "rubocop-rake", version: []),
+      GemInstaller::Spec.new(name: "rubocop-md", version: []),
+      GemInstaller::Spec.new(name: "rubocop-sequel", version: []),
     ].freeze
```
Comment (ybiquitous) [inline lib/runners/processor/haml_lint.rb]: Can we share this gem list between `rubocop.rb` and `haml_lint.rb`?

Comment (yykamei) [reply lib/runners/processor/haml_lint.rb]: Yes, of course. I want to think about it later. It

Comment (yykamei) [reply lib/runners/processor/haml_lint.rb]: Do you have any plans? Honestly, I have no ideas about that.

Comment (ybiquitous) [reply lib/runners/processor/haml_lint.rb]: Here is my current idea:

```ruby
class Processor::HamlLint < Processor
  OPTIONAL_GEMS = [
    *Processor::RuboCop::OPTIONAL_GEMS,
    # additional gems for HAML-Lint
  ]
end
```

Comment (yykamei) [reply lib/runners/processor/haml_lint.rb]: Thank you, if I didn't ask you about that, I might implement another solution, which extracts `OPTIONAL_GEMS` and declare it as a module. It's annoying for me, and I hesitate to do it.

Comment (yykamei) [reply lib/runners/processor/haml_lint.rb]: 31c2b91

Comment (yykamei) [reply lib/runners/processor/haml_lint.rb]: OK, I will fix to make the CI pass.

Comment (yykamei) [reply lib/runners/processor/haml_lint.rb]: What do you think about this fix? e415bd6

File: lib/runners/processor/haml_lint.rb [hunk 2]
```diff
@@ -26,34 +28,11 @@ class Processor::HamlLint < Processor
       )
     end
 
-    # DEPRECATED: Implicit dependencies
     # @see https://github.com/sider/runner_rubocop/blob/3.1.0/lib/entrypoint.rb#L27-L52
```
Comment (ybiquitous) [inline lib/runners/processor/haml_lint.rb]: By the way, can we remove the comment?

```suggestion
```

File: lib/runners/processor/haml_lint.rb (hunk 1 above)
Comment (ybiquitous) [reply lib/runners/processor/haml_lint.rb]: Is it necessary? `lib/runners.rb` requires all `*.rb` files within `lib/`.

File: lib/runners/processor/haml_lint.rb [hunk 3]
```diff
@@ -26,34 +28,11 @@ class Processor::HamlLint < Processor
...
-      GemInstaller::Spec.new(name: "fincop", version: []),
-      GemInstaller::Spec.new(name: "rubocop-github", version: []),
-      GemInstaller::Spec.new(name: "ezcater_rubocop", version: []),
-      GemInstaller::Spec.new(name: "rubocop-rspec", version: []),
-      GemInstaller::Spec.new(name: "rubocop-cask", version: []),
-      GemInstaller::Spec.new(name: "rubocop-thread_safety", version: []),
-    ].freeze
+      *Processor::RuboCop::OPTIONAL_GEMS,
+      # additional gems for HAML-Lint
+    ]
```
Comment (ybiquitous) [inline lib/runners/processor/haml_lint.rb]: [nitpick] For consistency and safety.

```suggestion
    ].freeze
```

File: lib/runners/processor/haml_lint.rb (hunk 1 above)
Comment (yykamei) [reply lib/runners/processor/haml_lint.rb]: I thought, but the CI will fail without this. Perhaps, the order of `require` is important in `lib/runners.rb`.

Comment (ybiquitous) [reply lib/runners/processor/haml_lint.rb]: Make sense. 👍

**Pull request merged**